src/domains/sign/env/prod/terraform.tfvars

```diff
@@ -30,7 +30,8 @@ subnets_cidrs = {
 }
 
 storage_account = {
-  enable_versioning             = false
+  enable_versioning             = true
+  change_feed_enabled           = true
```

Comment:
why enabling change feed?

Reply:
It is required to enable object replication